docs.yaml

```diff
@@ -64,6 +68,8 @@ rewrites:
   - https://www.h5l.org: https://github.com/heimdal/heimdal # Temporary until h5l.org is back online
   - http://docs.datastax.com/en/developer/cpp-driver/latest/: / # Get current version documentation
   - https://github.com/datastax/cpp-driver/blob/master/vc_build.bat: https://github.com/datastax/cpp-driver/blob/2.14.0/vc_build.bat
+  - search: 'https://community.datastax.com/index.html'
+    replace: 'https://www.datastax.com/workshops'
```

Comment:
Something doesn't seem right here; we also have this exact same URL (https://community.datastax.com/index.html) listed in the excludes above.  Seems like we should use one or the other.

Reply:
In related news: I got an email from @msmygit earlier today indicating that we should use https://cassandra.apache.org/_/community.html as the reference point instead of this.  I'm not sure which we actually want to use in this case... maybe @msmygit can clarify?

Reply:
Yes, we have already decommissioned https://community.datastax.com and are now sending folks over to StackOverflow and StackExchange as highlighted in the OSS Apache Cassandra community page that Bret has above.